recipes/wagtail-markdown/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "wagtail-markdown" %}
+{% set version = "0.4.1" %}
+{% set sha256 = "89b24b04d874feb97adb0299b5fbc06ea29b5a835ef765b82846575a1d77f63f" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/torchbox/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+    - Markdown ==2.6.2
+    - bleach ==1.4.2
+
+test:
+  imports:
+    - wagtailmarkdown
+
+about:
+  home: https://github.com/torchbox/wagtail-markdown
+  license: zlib
+  license_file: LICENSE
+  summary: 'Markdown support for Wagtail'
+  description: |
+     Markdown support for Wagtail
+  doc_url: https://github.com/torchbox/wagtail-markdown
+  dev_url: https://github.com/torchbox/wagtail-markdown
+
+extra:
+  recipe-maintainers:
+    - cshaley
+    - sannykr
```

Comment:
Guessing this is ok with you @cshaley. 😉

Might be worth adding a comment in issue ( https://github.com/conda-forge/staged-recipes/issues/139 ) noting what @sannykr can add you as a maintainer on generally. 😄

Reply:
Yes, it's okay, and done.